tests/testthat/_snaps/dependencies.md

```diff
@@ -0,0 +1,17 @@
+# renv warns when parsing dependencies from a folder with too many files
+
+    Code
+      dependencies(errors = "fatal")
+    Output
+      Finding R package dependencies ... Done!
+      WARNING: One or more problems were discovered while enumerating dependencies.
+      
+      renv-test-ecd07ec4e426
+      ----------------------
+      
+      ERROR 1: directory "renv-test-ecd07ec4e426" contains 28 files; consider ignoring this directory
```

Comment:
I didn't look into this, but it's suspicious that the test writes 26 files and this reports 28.

Reply:
The extra files are `dependencies.R` and `renv`, which are not suspicious 😄